tsconfig.json

```diff
@@ -1,18 +1,18 @@
-{
-  // tsconfig for editor must include both src and tests
-  "compilerOptions": {
-    "noEmit": true,
-    "rootDir": ".",
-    "strict": true,
-    "declaration": true,
-    "noEmitHelpers": true,
-    "importHelpers": true,
-    "sourceMap": true,
-    "module": "CommonJS",
-    "target": "ES5",
-    "lib": ["ES2015"],
-    "types": ["node", "jest"],
-    "jsx": "preserve"
-  },
-  "include": ["src", "tests"]
-}
+{
+  // tsconfig for editor must include both src and tests
+  "compilerOptions": {
+    "noEmit": true,
+    "rootDir": ".",
+    "strict": true,
+    "declaration": true,
+    "noEmitHelpers": true,
+    "importHelpers": true,
+    "sourceMap": true,
+    "module": "CommonJS",
+    "target": "ES2015",
```

Comment:
Not sure if relevant but the change from ES5 to ES2015 is a breaking change :)

ES5 is not ES2015. ES2015 is ES6.

Reply:
If you see supported versions of node.js by eslint, I think it can be even higher

Reply:
You're right. I just saw the target change and figured I'd point it out :)

But given that ESLint 8 only supports `"node": "^12.22.0 || ^14.17.0 || >=16.0.0"` then I guess it's safe to use ES2019
https://node.green/#ES2019